scripts/sysdig-probe-loader

```diff
@@ -192,12 +192,24 @@ load_kernel_probe() {
 	URL=$(echo "${SYSDIG_PROBE_URL}/${SYSDIG_REPOSITORY}/sysdig-probe-binaries/${SYSDIG_PROBE_FILENAME}" | sed s/+/%2B/g)
 
 	echo "* Trying to download precompiled module from ${URL}"
-	if curl --create-dirs "${SYSDIG_PROBE_CURL_OPTIONS}" -o "${HOME}/.sysdig/${SYSDIG_PROBE_FILENAME}" "${URL}"; then
+
+	curl_out=$(curl --create-dirs "${SYSDIG_PROBE_CURL_OPTIONS}" -o "${HOME}/.sysdig/${SYSDIG_PROBE_FILENAME}" "${URL}" 2>&1)
+	if [ "$?" = "0" ]; then
 		echo "Download succeeded, loading module"
 		insmod "${HOME}/.sysdig/${SYSDIG_PROBE_FILENAME}"
 		exit $?
 	else
-		echo "Download failed, consider compiling your own ${PROBE_NAME} and loading it or getting in touch with the sysdig community"
+		if [[ "$curl_out" =~ "404 Not Found" ]]; then
+			# The curl output throws a "curl: (22) The requested URL returned error: 404 Not Found" - but that is because the probe doesn't exist in the repo.
+			echo "Download of ${PROBE_NAME} for version ${SYSDIG_VERSION} failed. This is because the probe for this particular version does not exist in the repo."
```

Comment:
I really like that we're making this more customer facing. This is the first failure people run into a lot of the time.

Can you paste all of the output from the script into here?

I want it to clearly say "I tried to load it and that failed then I tried to build it and that failed and then I tried to download it and that failed. Here are your options: blah blah"

Reply:
These steps are already pretty straight-forwardly documented at each stage. Putting them all here again would be repetitive.